gammapy/stats/tests/test_feldman_cousins.py

```diff
@@ -0,0 +1,35 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import numpy as np
+from numpy.testing import assert_allclose
+from astropy.tests.helper import pytest
+
+from ...stats import (
+    fc_find_confidence_interval_gauss,
+    fc_find_confidence_interval_poisson,
+    fc_construct_confidence_belt_pdfs,
+    fc_get_upper_and_lower_limit,
+    fc_fix_upper_and_lower_limit,
+    fc_find_limit,
+    fc_find_average_upper_limit,
+    fc_construct_confidence_belt,
+)
+
```

Comment:
@dlennarz – This test currently fails like this:
https://travis-ci.org/gammapy/gammapy/jobs/75294485#L1007

Since scipy is an optional dependency you have to mark the test like this:
http://astropy.readthedocs.org/en/latest/development/testguide.html#tests-requiring-optional-dependencies